### DISCUSSION TITLE
10–14 October

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#105] [Progress Report]
🚴🏻 **Progress**

- #102 (which I introduced last week)
- #103 

😖 **Regressions**

- Brushing on the map seems to be causing an infinite loop

💬 **To Discuss**

None

📆 **Next Week**

- Fix brushing on the map
- Add brushing to the histogram
- Loading indicator

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Got time to work on #80 

😖 **Regressions**

- Poetry had issues detecting the correct Python version in CI

💬 **To Discuss**

- None

📆 **Next Week**

- Continue work on #80
